docs/VLSI/Sky130-OpenROAD-Tutorial.rst

```diff
@@ -0,0 +1,220 @@
+
+.. code-block:: shell
+
+    make verilog tutorial=sky130-openroad
+
+The ``CONFIG=TinyRocketConfig`` selects the target generator config in the same manner as the rest of the Chipyard framework. This elaborates a stripped-down Rocket Chip in the interest of minimizing tool runtime. The resulting verilog is located in ``./generated-src/chipyard.TestHarness.TinyRocketConfig/chipyard.TestHarness.TinyRocketConfig.top.v``.
+
+Note that in the generated Verilog, there are generic memory macros for the various memory components (dcache, icache, tag array, PTW). 
+This is the same Verilog that is generated for RTL simulations in the ``~chipyard/sims/verilator`` directory, see :ref:`Simulation/Software-RTL-Simulation:Software RTL Simulation` for directions on how to run these simulations.
+
+Building the Design
+^^^^^^^^^^^^^^^^^^^
+To set up all prerequisites for the build system to push the design and SRAM macros through the flow:
+
+.. code-block:: shell
+
+    make buildfile tech_name=sky130 TOOLS_CONF=example-openroad.yml CONFIG=TinyRocketConfig
+
+The command ``make buildfile`` generates a set of Make targets in ``build/hammer.d``. 
+It needs to be re-run if environment variables are changed. 
+It is recommended that you edit these variables directly in the Makefile rather than exporting them to your shell environment.
+
+
+Synthesis
+^^^^^^^^^
+
+.. code-block:: shell
+
+    make syn tutorial=sky130-openroad
+
+Post-synthesis logs and collateral are in ``build/syn-rundir``. 
+
+.. The raw quality of results data is available at ``build/syn-rundir/reports``, and methods to extract this information for design space exploration are a work in progress.
+
+Place-and-Route
+^^^^^^^^^^^^^^^
+.. code-block:: shell
+
+    make par tutorial=sky130-openroad
+
+After completion, the final database can be opened in an interactive OpenROAD session.
+
+.. code-block:: shell
+
+    cd ./build/par-rundir
+    ./generated-scripts/open_chip
+
+
+Below is the post-PnR layout for the TinyRocketConfig in Sky130 generated by OpenROAD.
+
+.. image:: ../_static/images/vlsi-openroad-par-tinyrocketconfig.png
+
+Intermediate databases are written in ``build/par-rundir`` between each step of the ``par`` action. These databases can be restored in an interactive OpenROAD session as desired for debugging purposes.
+
+.. code-block:: shell
+
+    openroad  # launch OpenROAD tool
+    openroad> read_db pre_global_route
+
+.. Timing reports are found in ``build/par-rundir/timingReports``. They are gzipped text files.
+
+DRC & LVS
+^^^^^^^^^
+To run DRC & LVS:
+
+.. code-block:: shell
+
+    make drc tutorial=sky130-openroad
+    make lvs tutorial=sky130-openroad
+
+Some DRC errors are expected from this PDK, especially with regards to the SRAMs, as explained in the 
+`Sky130 Hammer plugin README  <https://github.com/ucb-bar/hammer/blob/master/src/hammer-vlsi/technology/sky130/README.md>`__.
+
+
+VLSI Flow Control
```

Comment:
I like this new section, though I think you can put the bulk of it into section 5.2 Core Hammer (after Hooks, perhaps) and then note that users should consult the plugin for the list of steps.